zerver/openapi/zulip.yaml

```diff
@@ -19958,7 +19979,43 @@ paths:
                 message_retention_days:
                   $ref: "#/components/schemas/MessageRetentionDays"
                 can_remove_subscribers_group:
-                  $ref: "#/components/schemas/CanRemoveSubscribersGroupId"
+                  description: |
+                    The set of users who have permission to unsubscribe others from this
+                    channel expressed as an [update to a group-setting value][update-group-setting].
+
+                    Note that a user who is a member of the specified user group must
+                    also [have access](/help/channel-permissions) to the channel in
+                    order to unsubscribe others.
+
+                    **Changes**: Prior to Zulip 10.0 (feature level 320), this value used
+                    to be of type integer and did not accept anonymous user groups.
```

Comment:
I think it'd be clearer to write it this way -- rather than saying what it didn't accept, we just say clearly what it does accept, using a term that appears extensively in the documentation:

> Prior to Zulip ..., this value was always the integer ID of a system group.

Reply:
Updated.